pure.zsh

```diff
@@ -78,8 +78,23 @@ prompt_pure_check_git_arrows() {
 	arrow_status=(${(ps:\t:)arrow_status})
 	local arrows left=${arrow_status[1]} right=${arrow_status[2]}
 
-	(( ${right:-0} > 0 )) && arrows+="${PURE_GIT_DOWN_ARROW:-⇣}"
-	(( ${left:-0} > 0 )) && arrows+="${PURE_GIT_UP_ARROW:-⇡}"
+	if (( ${right:-0} > 0 )); then
+        commits=$(command git rev-list --count HEAD..@{upstream} 2>/dev/null)
```

Comment:
This information is already available in `$right`, no need for a re-executing `git rev-list`.